common/setup.go

```diff
@@ -106,6 +92,15 @@ func setupCluster(cfg *config.Config) (err error) {
 // useKubeconfig reads the path provided for a TEST_KUBECONFIG and uses it for testing.
 func useKubeconfig(cfg *config.Config) (err error) {
 	filename := string(cfg.Kubeconfig)
+
+	_, err = clientcmd.RESTConfigFromKubeConfig(cfg.Kubeconfig)
```

Comment:
The kubeconfig stuff mentioned above. This is checking "hey is this kubeconfig a valid kubeconfig? no? okay then try reading it as a file. yes? sweet return."